apps/metadata-utils/src/types.ts

```diff
@@ -104,3 +104,10 @@ export type columnValue = string | number | boolean | columnValueObject;
 interface columnValueObject {
   [x: string]: columnValue;
 }
+
+export type IInputValue = string | number | boolean;
+
+export type IInputValueLabel = {
```

Comment:
can we please merge this type with the same time for checkboxgroup and radiogroup?